### PR TITLE
Added Multiple Personality Disorder

### PR DESCRIPTION
**Pull request by Budcoderr**

### CODE REVIEW

**Review by icyflame — COMMENTED** [review]
- fix grammar in the paragraph
- reduce the length of oneliner about MPD

**Review by icyflame — CHANGES_REQUESTED** [review]
- fix grammar in the paragraph
- reduce the length of oneliner about MPD

**Comment by Budcoderr** [comment]
Changes done. Please review

File: readme.md [hunk 1]
```diff
@@ -176,6 +176,17 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+#### [Multiple Personality Disorder](wikipedia.org/Dissociative_identity_disorder)
+
+>Condition in which distinct identities alternately take control of—an individual
```
Comment (icyflame) [inline readme.md:181]: space

File: readme.md [hunk 2]
```diff
@@ -176,6 +176,17 @@ the anecdote, you can throw away the data."
...
+
+In 1791, the first detailed account of "exchanged personality" was written about a 20-year-old German woman. DID was majorly studied between 1880 and 1920 and in 1944. According to Mosby's Medical Dictionary it is defined  as, "an unconscious defence mechanism by which an idea, thought, emotion, or other mental process is separated from the consciousness and thereby loses emotional significance."
```
Comment (icyflame) [inline readme.md:183]: What is DID? This whole paragraph lacks context

**Review by icyflame — CHANGES_REQUESTED** [review]
Please proof read entries before submitting

**Comment by icyflame** [comment]
Closing this for now. Please re-open after making the request changes.

**Pull request closed without merging**